src/apprt/gtk/App.zig

```diff
@@ -36,8 +36,7 @@ const c = @import("c.zig").c;
 const version = @import("version.zig");
 const inspector = @import("inspector.zig");
 const key = @import("key.zig");
-const x11 = @import("x11.zig");
-const wayland = @import("wayland.zig");
+const platform = @import("platform.zig");
```

Comment:
I'll just put this right at the beginning since it's everywhere: I'm not sure if "platform" is the right noun for this. Platform is a catch-all of a pluggable thing (similar to widget or plugin or extension). But I feel like in this case there's probably a better noun to describe _what_ we're actually abstracting. 

For example, is it:

  * Windowing protocol
  * Display server
  * etc.

(Making those one word is harder...)

But this is pretty core to things so I'd like to make sure we get this right.

Reply:
I like "protocol". Calling it "backend" like GTK/GDK does feels *okay* though that might be confused for app runtimes